Skate_Behaviour_IO.m

```diff
@@ -39,6 +41,12 @@
 nFalseAlarm = 7;
 nCorrectResponse = 9;
 
+pStandard = 1;
```

Comment:
whats the difference between p and n target, make this clear using a better name, and it is better to note make new variables for just some subjects

Reply:
That set of variables: pTarget, pFalseAlarm, etc... is meant to read the triggers from the newer set of subjects. For this new set of subjects, 1 is the number for standards, 2 is target, etc... in the original analysis and with most of our sample, nTarget is 5, nStandard is 3, etc. That's why I generated the dif_trig variable for the new subjects with the different triggers.

Reply:
right... but there is no need to use different variable names , just assign those variables new numbers for those subjects, then you don't need all the or statments lower in the code, since they will look for the same variable

Reply:
I got rid of the extra variables and extra or statements. 

I also tested that the loop I wrote works correctly.  see below two snapshots of the command window showing both set of triggers being added under the same script. 

![image](https://user-images.githubusercontent.com/38795068/56007700-c9ae3d80-5c96-11e9-88e5-8785835ac78c.png)
![image](https://user-images.githubusercontent.com/38795068/56007709-d6cb2c80-5c96-11e9-8ca6-741b271b2922.png)